spec/lib/bank/account_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'bank'
+
+describe Bank::Account do
+
+  before :each do
+    @bank = Bank::Account.new("Checking", 50)
+  end
+
+  describe "#new" do
+    it "creates a new instance" do
+      expect(Bank::Account).to respond_to(:new)
+    end
+
+    describe "initialize" do
+      it "assigns the id instance variable" do
+        expect(@bank.id).to eq("Checking")
+      end
+
+      it "assigns the balance instance variable" do
+        expect(@bank.balance).to eq(50)
+      end
+
+      it "sets balance to an integer" do
```

Comment:
In this case, you should split these into 2 tests, once for accepting a number, and once for accepting a nonnumber but still having balance 0.